sqlmesh/core/test/definition.py

```diff
@@ -18,6 +18,7 @@ class TestError(SQLMeshError):
 
 
 class ModelTest(unittest.TestCase):
+    __test__ = False
```

Comment:
Pytest discovery was picking up SqlModelTest as a test case below and breaking our unit tests.